### PR TITLE
Some management module cleanups

### PR DESCRIPTION
**Pull request by mathieucarbou**

### CODE REVIEW

File: management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatistics.java [hunk 1]
```diff
@@ -221,17 +220,21 @@ public void dispose() {
 
 
   @SuppressWarnings({"unchecked", "rawtypes"})
-  private ConcurrentMap<String, OperationStatistic<?>> discoverCountStatistics() {
-    ConcurrentHashMap<String, OperationStatistic<?>> result = new ConcurrentHashMap<String, OperationStatistic<?>>();
```
Comment (anthonydahanne) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatistics.java]: no more putIfAbsent ?

Comment (mathieucarbou) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatistics.java]: not in a normal map in jdk6

Comment (anthonydahanne) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatistics.java]: ok, what I meant is : why the switch from Concurrent to normal ?

Comment (mathieucarbou) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatistics.java]: Why would you use a ConcurrentMap here, that, besides, depend on the internal impl of Ehcache ? It's just a local map which is created sequentially, then read-only.

Comment (anthonydahanne) [inline management/src/main/java/org/ehcache/management/providers/statistics/EhcacheStatistics.java]: haha, true, put I liked the conciseness of putIfAbsent :-)

**Pull request merged**